tests/e2e/project/browser.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { test, expect } from "@tests/support/fixtures.js";
```

Comment:
I think we have most source browsing tests in `project.spec.ts`. I'd move these tests there.

Reply:
1ea4b49d